website/docs/docs/cloud/git/import-a-project-by-git-url.md

```diff
@@ -138,6 +138,10 @@ If you use Azure DevOps and you are on the dbt Cloud Enterprise plan, you can im
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/d19f199-Screen_Shot_2020-03-09_at_4.13.50_PM.png" title="Enter and save the public key generated for your repository by dbt Cloud" />
 
+import GitProvidersCI from '/snippets/_git-providers-supporting-ci.md';
```

Comment:
same comment as below:

i suggest placing the "Availability of features by Git provider" section right after "Git protocols" but before "Managing deploy keys" that way, it balances visibility and relevance while maintaining a logical progression.

currently, i think it's placed in a weird place and breaks up the git provider listing

<img width="244" alt="Screenshot 2025-01-10 at 11 26 20" src="https://github.com/user-attachments/assets/6f3d0719-9b15-4fb0-99e4-2e5e89ac3aec" />